src/components/sharedComponents/TokenLogo.tsx

```diff
@@ -94,7 +94,7 @@ const TokenLogo: FC<TokenLogoProps> = ({ size = 24, token }) => {
 
   useEffect(() => {
     setHasError(false)
-  }, [logoURI])
+  }, [])
```

Comment:
I'm not sure about this change, as it will affect the intended behavior.

Clearly, this is a smelly code, but I think this change will affect the current behavior.